custom-targets/vertex-ai/quickstart/QUICKSTART.md

```diff
@@ -0,0 +1,200 @@
+   ```
+
+If this is your first time using Vertex AI in this project, this operation will
+take 5 minutes or so.
+
+The endpoint id will be used to refer to the endpoint, rather than the display name.
+
+## 5. Create delivery pipeline, target, and custom target type
+
+Replace placeholders in `clouddeploy.yaml` with actual values:
+```shell
+sed -i "s/\$PROJECT_ID/${PROJECT_ID}/g" clouddeploy.yaml
+sed -i "s/\$REGION/${REGION}/g" clouddeploy.yaml
+sed -i "s/\$ENDPOINT_ID/${ENDPOINT_ID}/g" clouddeploy.yaml
+```
+Apply the Cloud Deploy configuration defined in `clouddeploy.yaml`:
+
+```shell
+gcloud deploy apply --file=clouddeploy.yaml --project=$PROJECT_ID --region=$REGION
+```
+
+## 6. Install the Custom Target
+
+From within the `quickstart` directory, run this command to build the Vertex AI model deployer image and
+install the custom target resources:
+
+```shell
+../build_and_register.sh -p $PROJECT_ID -r $REGION
+```
+
+For information about the `build_and_register.sh` script, see the [README](../README.md#build)
+
+## 7. Create a release and rollout
+
+Create a Cloud Deploy release for the configuration defined in the `configuration` directory. This automatically
+creates a rollout that deploys the first model version to the target.
+
+```shell
+gcloud deploy releases create release-001 \
+    --delivery-pipeline=vertex-ai-cloud-deploy-pipeline \
+    --project=$PROJECT_ID \
+    --region=$REGION \
+    --source=configuration \
+    --deploy-parameters="vertexAIModel=projects/$PROJECT_ID/locations/$REGION/models/test_model"
+```
+
+### Explanation of command line flags
+
+The `--source` command line flag instructs gcloud where to look for the configuration files relative to the working directory where the command is run.
+
+The `--deploy-parameters` flag is used to provide the custom deployer with additional parameters needed to perform the deployment.
+
+Here, we are providing the custom deployer with two parameters:
+* `vertexAIModel`: the full resource name of the model to deploy
+* `vertexAIConfigPath`: location within the source directory where the per-target configuration folders are located
+
+The remaining flags specify the Cloud Deploy Delivery Pipeline. `--delivery-pipeline` is the name of
+the delivery pipeline where the release will be created, and the project and region of the pipeline
+is specified by `--project` and `--region` respectively.
+
+
+### Monitor the release's progress
+
+To check release details, run this command:
+
+```shell
+gcloud deploy releases describe release-001 --delivery-pipeline=vertex-ai-cloud-deploy-pipeline --project=$PROJECT_ID --region=$REGION
+```
+
+Run this command to filter only the render status of the release:
+
+```shell
+gcloud deploy releases describe release-001 --delivery-pipeline=vertex-ai-cloud-deploy-pipeline --project=$PROJECT_ID --region=$REGION --format "(renderState)"
+```
+
+## 8. Monitor rollout status
```

Comment:
How long is the post deploy? If it's short then I'd advocate for including it in the quickstart.